### ISSUE TITLE
Accessibility violation: Complex table does not have headers for each cell properly defined with 'header' or 'scope'

### ISSUE THREAD

**Issue by jose-h-pena** [carbon-design-system/carbon-components-vue#1402]
## Detailed description

> Describe in detail the issue you're having.

This issue is related to CvDataTable. Using IBM Accessibility Checker on the CvDataTable results on the following violation: "Complex table does not have headers for each cell properly defined with 'header' or 'scope'"

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?
CvDataTable

> What did you expect to happen? What happened instead? What would you like to see changed?

Was expecting no violations at the Accessibility Checker.

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

"@carbon/vue": "^2.42.0",

## Steps to reproduce the issue

1. create a notification
2. pass the Accessibility Checker at the notification page

## Additional information

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/122560662/212149965-2ec1402e-273a-4c1b-9243-ffc3c904aeb3.png">
